Gemfile.lock

```diff
@@ -267,170 +272,157 @@ GEM
       addressable (~> 2.3)
     less (2.6.0)
       commonjs (~> 0.2.7)
-    less-rails (2.7.1)
+    less-rails (2.8.0)
       actionpack (>= 4.0)
       less (~> 2.6.0)
       sprockets (> 2, < 4)
       tilt
     libv8 (3.16.14.13)
-    listen (3.0.6)
-      rb-fsevent (>= 0.9.3)
-      rb-inotify (>= 0.9.7)
+    listen (3.1.5)
+      rb-fsevent (~> 0.9, >= 0.9.4)
+      rb-inotify (~> 0.9, >= 0.9.7)
+      ruby_dep (~> 1.2)
     loofah (2.0.3)
       nokogiri (>= 1.5.9)
     lumberjack (1.0.10)
-    macaddr (1.7.1)
-      systemu (~> 2.6.2)
     mail (2.6.4)
       mime-types (>= 1.16, < 4)
-    mechanize (2.7.4)
-      domain_name (~> 0.5, >= 0.5.1)
-      http-cookie (~> 1.0)
-      mime-types (>= 1.17.2, < 3)
-      net-http-digest_auth (~> 1.1, >= 1.1.1)
-      net-http-persistent (~> 2.5, >= 2.5.2)
-      nokogiri (~> 1.6)
-      ntlm-http (~> 0.1, >= 0.1.1)
-      webrobots (>= 0.0.9, < 0.2)
-    meta-tags (2.1.0)
-      actionpack (>= 3.0.0)
+    meta-tags (2.3.1)
+      actionpack (>= 3.2.0, <= 5.1.0)
     method_source (0.8.2)
-    mime-types (2.99.2)
-    mimemagic (0.3.1)
+    mime-types (2.99.3)
+    mimemagic (0.3.2)
     mini_portile2 (2.1.0)
-    minitest (5.9.0)
-    momentjs-rails (2.11.0)
+    minitest (5.9.1)
+    momentjs-rails (2.15.1)
       railties (>= 3.1)
     multi_json (1.12.1)
     multipart-post (2.0.0)
-    mysql2 (0.4.2)
+    mysql2 (0.4.5)
     nenv (0.3.0)
-    net-http-digest_auth (1.4)
-    net-http-persistent (2.9.4)
     net-scp (1.2.1)
       net-ssh (>= 2.6.5)
     net-ssh (3.2.0)
-    nokogiri (1.6.8)
+    nio4r (1.2.1)
+    nokogiri (1.6.8.1)
       mini_portile2 (~> 2.1.0)
-      pkg-config (~> 1.1.7)
-    non-stupid-digest-assets (1.0.6)
+    non-stupid-digest-assets (1.0.8)
       sprockets (>= 2.0)
-    notiffany (0.0.8)
+    notiffany (0.1.1)
       nenv (~> 0.1)
       shellany (~> 0.0)
-    ntlm-http (0.1.1)
     orm_adapter (0.5.0)
-    paperclip-azure (0.2.14)
+    paperclip-azure (0.3.0)
       addressable
       azure
       hashie
-    parser (2.3.1.2)
+    parser (2.3.1.4)
       ast (~> 2.2)
-    pdf-reader (1.3.3)
+    pdf-reader (1.4.0)
       Ascii85 (~> 1.0.0)
-      afm (~> 0.2.0)
+      afm (~> 0.2.1)
       hashery (~> 2.0)
       ruby-rc4
       ttfunk
-    php-serialization (0.5.3)
+    php-serialization (1.0.0)
       racc
-    pkg-config (1.1.7)
-    polyamorous (1.3.0)
+    polyamorous (1.3.1)
       activerecord (>= 3.0)
     powerpack (0.1.1)
-    pry (0.10.3)
+    pry (0.10.4)
       coderay (~> 1.1.0)
       method_source (~> 0.8.1)
       slop (~> 3.4)
     racc (1.4.14)
-    rack (1.6.4)
-    rack-dev-mark (0.7.4)
+    rack (2.0.1)
+    rack-dev-mark (0.7.5)
       rack (>= 1.1)
     rack-test (0.6.3)
       rack (>= 1.0)
-    rails (4.2.6)
-      actionmailer (= 4.2.6)
-      actionpack (= 4.2.6)
-      actionview (= 4.2.6)
-      activejob (= 4.2.6)
-      activemodel (= 4.2.6)
-      activerecord (= 4.2.6)
-      activesupport (= 4.2.6)
+    rails (5.0.0)
```

Comment:
__[brakeman]__
* Rails 5.0.0 content_tag does not escape double quotes in attribute values (CVE-2016-6316). Upgrade to 5.0.0 [:ref](https://groups.google.com/d/msg/ruby-security-ann/8B2iV2tPRSE/JkjCJkSoCgAJ)

<!-- Comment by [SideCI](https://sideci.com) -->